sematic/api/endpoints/tests/test_auth.py

```diff
@@ -132,18 +136,40 @@ def test_login_invalid_domain(test_client: flask.testing.FlaskClient):  # noqa:
     with mock_server_settings(
         {
             ServerSettingsVar.GOOGLE_OAUTH_CLIENT_ID: "ABC123",
-            ServerSettingsVar.SEMATIC_AUTHORIZED_EMAIL_DOMAIN: "example.com",
+            ServerSettingsVar.SEMATIC_AUTHORIZED_EMAIL_DOMAIN: (
+                "example1.com,example2.com"
+            ),
         }
     ):
         with mock.patch(
             "google.oauth2.id_token.verify_oauth2_token",
-            retur_value={"hd": "wrong.domain"},
+            return_value={"hd": "wrong.domain"},
         ):
             response = test_client.post("/login/google", json={"token": "abc"})
 
             assert response.status_code == HTTPStatus.UNAUTHORIZED
 
 
+def test_login_valid_domain(
```

Comment:
Might not be pertinent to this change, but please consider adding a test case where an attacker tries to use a prefix email address.